fml/log_level.h

```diff
@@ -13,8 +13,11 @@ typedef int LogSeverity;
 constexpr LogSeverity kLogInfo = 0;
 constexpr LogSeverity kLogWarning = 1;
 constexpr LogSeverity kLogError = 2;
-constexpr LogSeverity kLogFatal = 3;
-constexpr LogSeverity kLogNumSeverities = 4;
+// A log that is not an error, is important enough to display even if ordinary
+// info is hidden.
+constexpr LogSeverity kLogImportant = 3;
```

Comment:
Will this break users who set a log level on the command line? If so, should we add a note in our release notes?

Reply:
Also,

If thew intent

Reply:
It shouldn't - I'm replacing the `FATAL` level with `IMPORTANT` - it's not actually possible to ignore FATAL, but I suppose someone who was suppressing `ERROR` will now get `IMPORTANT`. I doubt this is happening much if at all though.